test/software/amazon/ion/EquivTimelineTest.java

```diff
@@ -51,4 +53,19 @@ protected void checkEquivalence(IonValue left, IonValue right,
         assertEquals("calendar", lTime.calendarValue(), rTime.calendarValue());
         assertEquals("date", lTime.dateValue(), rTime.dateValue());
     }
+
+    @Override
+    public void roundTripEquivalence(IonDatagram input, boolean myExpectedEquality) throws IOException {
+        IonDatagram[] data = roundTripDatagram(input);
+        for(int i = 0; i < data.length - 1; i++){
+            runEquivalenceChecks(data[i], myExpectedEquality);
+            for(int j = i + 1; j < data.length; j++) {
+                for(int sexpIndice = 0; sexpIndice < data[i].size(); sexpIndice++) {
+                    for(int timestampIndice = 0; timestampIndice < ((IonSexp)data[i].get(sexpIndice)).size(); timestampIndice++) {
```

Comment:
In addition to `IonSexp`, these elements could also be `IonList`s.

Reply:
is one preffered?

Reply:
No, we just have to support both. Currently this will fail if they're `IonList`s.